src/e-pr.js

```diff
@@ -3,11 +3,55 @@
 const childProcess = require('child_process');
 const fs = require('fs');
 const path = require('path');
+const querystring = require('querystring');
 
+const got = require('got');
 const open = require('open');
 const program = require('commander');
 
 const evmConfig = require('./evm-config');
+const { color, fatal } = require('./utils/logging');
+
+// Adapted from https://github.com/electron/clerk
+function findNoteInPRBody(body) {
+  const onelineMatch = /(?:(?:\r?\n)|^)notes: (.+?)(?:(?:\r?\n)|$)/gi.exec(body);
+  const multilineMatch = /(?:(?:\r?\n)Notes:(?:\r?\n+)((?:\*.+(?:(?:\r?\n)|$))+))/gi.exec(body);
+
+  let notes = null;
+  if (onelineMatch && onelineMatch[1]) {
+    notes = onelineMatch[1];
+  } else if (multilineMatch && multilineMatch[1]) {
+    notes = multilineMatch[1];
+  }
+
+  if (notes) {
+    // Remove the default PR template.
+    notes = notes.replace(/<!--.*?-->/g, '');
+  }
+
+  return notes ? notes.trim() : notes;
+}
+
+async function getPullRequestNotes(pullNumber) {
+  let notes = null;
+
+  const opts = {
+    url: `https://api.github.com/repos/electron/electron/pulls/${pullNumber}`,
+    responseType: 'json',
+    throwHttpErrors: false,
+  };
+  try {
+    const response = await got(opts);
+    if (response.statusCode !== 200) {
+      fatal(`Could not find PR: ${opts.url} got ${response.headers.status}`);
```

Comment:
i'd rather `console.log` this with an error and not throw an error with a stack i think 🤔 

```js
console.log(color.err, Could not find PR: ${opts.url} got ${response.headers.status});
```

Reply:
I think that's actually functionally what's happening with the current code. It looks like `fatal` will only include a stack if an error is provided, if a string is used (like here), then it will do what you suggested, just using `console.error` instead:

```javascript
function fatal(e) {
  if (typeof e === 'string') {
    console.error(`${color.err} ${e}`);
  } else {
    console.error(`${color.err} ${e.stack ? e.stack : e.message}`);
  }
  process.exit(1);
}
```

Reply:
ope wait you're totally right, i forgot we changed that!